lib/rails/perftest/railties/testing.tasks

```diff
@@ -4,14 +4,25 @@ namespace :test do
   task 'perftest:benchmark_mode' do
     ENV["BENCHMARK_TESTS"] = '1'
   end
+  if Rake::Task.task_defined?('test:prepare')
```

Comment:
Could the `if` statement be used to just determine the tasks dependencies? This would get us around copying everything.

```ruby
dependencies = if ...

Rails::SubTestTask.new(benchmark: [*dependencies, 'test:perftest:benchmark_mode']) do |t|
  t.libs << 'test'
  t.pattern = 'test/performance/**/*_test.rb'
end
```